microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -219,16 +433,44 @@ See also:
 
 Use this API to check the status of the publish operation.
 
+##### [v1.1](#tab/v1-1)
+
+<!-- todo: update for 1.1 -->
```

Comment:
h2 # 4: update to v1.1 code
@yvsubhash 
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3269&tabs=v1#checking-the-publishing-status

Reply:
v1.1 code in tabs is ready for review.  closing.